integration-cli/docker_cli_plugins_test.go

```diff
@@ -3,14 +3,18 @@ package main
 import (
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/go-check/check"
+
+	"strings"
+)
+
+var (
+	name        = "tiborvass/no-remove"
```

Comment:
These top-level variable names need to be a bit more specific since this is being set for the whole package.